.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,42 +1,19 @@
-# All SDK Contribution checklist:
+# Contributing to the SDK
 
-This checklist is used to make sure that common guidelines for a pull request are followed.
-- [ ] **Please open PR in `Draft` mode if it is:**
-	- Work in progress or not intended to be merged.
-	- Encountering multiple pipeline failures and working on fixes.
-- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
-- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md).**
-- [ ] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**
+Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repo or have questions.
 
-### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
-- [ ] Title of the pull request is clear and informative.
-- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
+## Pull Request etiquette and best practices
```

Comment:
This was distilled from comments during our team discussion today

Reply:
I think all this text would be best in CONTRIBUTING.md. We could link to the bookmark directly.

Repeated studies show people don't like reading large blocks of text. I suspect - after seeing it more often in repos with larger template - people just `Ctrl+A + Del` when they see large templates like this with descriptive text just so they can write what they need. For example, have you ever seen an Azure/azure-rest-api-specs PR template filled out?

Reply:
It's a good discussion. Arguably we should just anchor link to a few key parts, this being one of them.

Reply:
Moved it to the main CONTRIBUTING.md